apps/wrf/install_wps_mvapich2.sh

```diff
@@ -23,7 +26,7 @@ module load gcc-9.2.0
 
 export HDF5=$(spack location -i hdf5^mvapich2)
 export NETCDF=$(spack location -i netcdf-fortran^mvapich2)
-export WRF_DIR=${SHARED_APP}/WRF-${WRF_VERSION}
+export WRF_DIR=${SHARED_APP}/${SKU_TYPE}/wrf-mvapich2/WRF-${WRF_VERSION}
```

Comment:
Can't the APP_DIR be reused here ?
Is there any differences between WRF_VERSION and APP_VERSION ?

Reply:
The WPS installation needs the path to WRF and so APP_DIR cannot be used (it is location of WPS and not WRF).
APP_VERSION is WPS version (4.1), but I also need WRF version (WRF_VERSION) which is different, 4.1.3.